README.md

```diff
@@ -352,6 +370,21 @@ Remove the line to DROP INDEX.
 
 Remove the line to DROP CONSTRAINT.
 
+### CREATE (OR REPLACE) VIEW
+
+```diff
+ CREATE OR REPLACE VIEW foo AS
+   select u.id as id, p.id as post_id
+   from  (
+     mysqldef_test.users as u
+     join mysqldef_test.posts as p on ((u.id = p.user_id))
+   )
+ ;
++ CREATE OR REPLACE VIEW foo AS select u.id as id, p.id as post_id from (mysqldef_test.users as u join mysqldef_test.posts as p on (((u.id = p.user_id) and (p.is_deleted = 0))));
```

Comment:
```suggestion
+ CREATE OR REPLACE VIEW foo AS select u.id as id, p.id as post_id from (psqldef_test.users as u join psqldef_test.posts as p on (((u.id = p.user_id) and (p.is_deleted = 0))));
```

because this is an example for PostgreSQL.

Reply:
Or, according to the tests, it seems a database name is not needed for PostgreSQL.
```suggestion
+ CREATE OR REPLACE VIEW foo AS select u.id as id, p.id as post_id from (users as u join posts as p on (((u.id = p.user_id) and (p.is_deleted = 0))));
```

Reply:
Oh... I'm so sorry about that 🙇